har/har_handlers.go

```diff
@@ -67,9 +68,15 @@ func (h *resetHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		log.Errorf("har: method not allowed: %s", req.Method)
 		return
 	}
-	h.logger.Reset()
 
-	log.Infof("resetHandler.ServeHTTP: HAR logs cleared")
+	if isBool, _ := strconv.ParseBool(req.URL.Query().Get("return")); isBool {
```

Comment:
`isBool` is a confusing name for this variable (it seems to represent whether the value is a bool but it's actually capturing the value of `return`)

just call it `b` or `v` - it's only relevant for 2 lines, so a single character is fine